pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/FileSystemUtils.java

```diff
@@ -33,18 +31,16 @@ private FileSystemUtils() {
      * <p>
      * Make a link using the system's ``ln`` command.
      * <p>
-     * @param project The project to run within.
      * @param target The target directory that the link points to.
      * @param destination The destination directory or the name of the link.
      * @throws IOException if symlink can't be made
      */
-    public static void makeSymLink(Project project, File target, File destination) throws IOException {
+    public static void makeSymLink(File target, File destination) throws IOException {
         /*
          * Check if the file exists because the link checking logic in Gradle differs
          * between Linux and OS X machines.
          */
-        if (!project.file(destination).exists()) {
-            Files.createSymbolicLink(destination.toPath(), target.toPath());
-        }
+        Files.deleteIfExists(destination.toPath());
```

Comment:
Wouldn't this remove users' files without their knowledge?
Seems destructive compared to previous implementation.